src/go/rpk/pkg/cli/cmd/cluster/config/export.go

```diff
@@ -0,0 +1,202 @@
+			Value: name,
+		}
+
+		var valueNode yaml.Node
+		switch x := curValue.(type) {
+		case int:
+			valueNode = yaml.Node{
+				Kind:  yaml.ScalarNode,
+				Value: strconv.Itoa(x),
+			}
+		case float64:
+			if x == float64(int64(x)) {
+				valueNode = yaml.Node{
+					Kind:  yaml.ScalarNode,
+					Value: strconv.FormatInt(int64(x), 10),
+				}
+			} else {
+				valueNode = yaml.Node{
+					Kind:  yaml.ScalarNode,
+					Value: strconv.FormatFloat(x, 'f', -1, 64),
+				}
+			}
+		case string:
+			valueNode = yaml.Node{
+				Kind:  yaml.ScalarNode,
+				Value: x,
+			}
+		case bool:
+			valueNode = yaml.Node{
+				Kind:  yaml.ScalarNode,
+				Value: strconv.FormatBool(x),
+			}
+		case []interface{}:
+			valueNode = yaml.Node{
+				Kind:  yaml.SequenceNode,
+				Value: "",
+			}
+			// TODO generalize for other types of array
+			for _, v := range x {
+				seqNode := yaml.Node{
+					Kind:  yaml.ScalarNode,
+					Value: v.(string),
+				}
+				valueNode.Content = append(valueNode.Content, &seqNode)
+			}
+		case nil:
+			valueNode = yaml.Node{
+				Kind:  yaml.ScalarNode,
+				Value: "",
+			}
+		default:
+			out.Die("Unexpected property value type: %s: %T", name, curValue)
+			continue
+		}
+
+		// TODO: nullability in description
+		// TODO: needs-restart in description
+
+		// Preface each property with a descriptive comment
+		if meta.Example != "" {
+			propertyNode.HeadComment = fmt.Sprintf("%s (e.g. '%s')", meta.Description, meta.Example)
+		} else {
+			propertyNode.HeadComment = meta.Description
+		}
+
+		// Blank line between each property for readability
+		propertyNode.FootComment = "\n"
+
+		yamlNode.Content = append(yamlNode.Content, &propertyNode)
+		yamlNode.Content = append(yamlNode.Content, &valueNode)
+	}
+
+	serialized, err := yaml.Marshal(&yamlNode)
+	if err != nil {
+		return err
+	}
+
+	_, err = file.Write(serialized)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func newExportCommand(fs afero.Fs, all *bool) *cobra.Command {
+	var filename string
+
+	cmd := &cobra.Command{
+		Use:   "export",
+		Short: "Export cluster configuration.",
+		Long: `Export cluster-wide configuration properties
+
+Writes out a YAML representation of the cluster configuration to a file,
+suitable for editing and later applying with the corresponding 'import'
+command.
+
+By default, low level tunables are excluded: use the '--all' flag
+to include all properties including these low level tunables.
+`,
+		Run: func(cmd *cobra.Command, args []string) {
+
+			p := config.ParamsFromCommand(cmd)
+			cfg, err := p.Load(fs)
+			out.MaybeDie(err, "unable to load config: %v", err)
+
+			client, err := admin.NewClient(fs, cfg)
+			out.MaybeDie(err, "unable to initialize admin client: %v", err)
+
+			// GET the schema
+			schema, err := client.ClusterConfigSchema()
+			out.MaybeDie(err, "unable to query config schema: %v", err)
+
+			// GET current config
+			var currentConfig admin.Config
+			currentConfig, err = client.Config()
```

Comment:
`currentConfig, err := client.Config()`

Reply:
I still see this -- can remove the previous `var currentConfig` and replace with `:=` on this line.